client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -611,7 +611,7 @@ class WpcomChecklistComponent extends PureComponent {
 				{ ...baseProps }
 				bannerImageSrc="/calypso/images/stats/tasks/launch.svg"
 				buttonText={ translate( 'Launch site' ) }
-				completedTitle={ translate( 'You launched your site' ) }
+				completedTitle={ translate( '' + 'You created your site' ) }
```

Comment:
What's the `'' +` for here?

Reply:
The million dollar question.

Don't remember doing that. Human error I guess. Or prettify gone beserk. Thanks @spen!